packages/nc-gui/components/workspace/CollaboratorsList.vue

```diff
@@ -56,67 +56,59 @@ onMounted(async () => {
+        <!-- <div class="flex w-2/5 text-gray-600">Added By</div> -->
+        <div class="flex w-2/5 text-gray-600">Action</div>
+      </div>
+      <div class="flex flex-col nc-scrollbar-md">
+        <div v-for="(collab, i) of filterCollaborators" :key="i"
+          class="relative w-full nc-collaborators nc-collaborators-list-row">
+          <div class="!py-0 w-2/5 email truncate">
             <div class="flex items-center gap-2">
               <span class="color-band" :style="{ backgroundColor: stringToColour(collab.email) }">{{
                 collab.email.slice(0, 2)
               }}</span>
-              {{ collab.email }}
+              <span class="truncate">
+                {{ collab.email }}
+              </span>
             </div>
-          </td>
-          <td class="text-gray-500 text-xs w-1/5 created-at">
+          </div>
+          <div class="text-gray-500 text-xs w-2/5 created-at truncate">
             {{ timeAgo(collab.created_at) }}
-          </td>
-          <td class="w-1/5 roles">
+          </div>
+          <div class="w-2/5 roles">
             <div class="nc-collaborator-role-select">
               <template v-if="accessibleRoles.includes(collab.roles)">
-                <RolesSelector
-                  :role="collab.roles"
-                  :roles="accessibleRoles"
-                  :description="false"
-                  :on-role-change="(role: WorkspaceUserRoles) => updateCollaborator(collab, role)"
-                />
+                <RolesSelector :role="collab.roles" :roles="accessibleRoles" :description="false"
+                  :on-role-change="(role: WorkspaceUserRoles) => updateCollaborator(collab, role)" />
               </template>
               <template v-else>
                 <RolesBadge class="!bg-white" :role="collab.roles" />
               </template>
             </div>
-          </td>
-          <td class="w-1/5">
-            <div class="-left-2.5 top-5">
-              <NcDropdown v-if="collab.roles !== WorkspaceUserRoles.OWNER" :trigger="['click']">
-                <MdiDotsVertical
-                  class="border-1 !text-gray-600 h-5.5 w-5.5 rounded outline-0 p-0.5 nc-workspace-menu transform transition-transform !text-gray-400 cursor-pointer hover:(!text-gray-500 bg-gray-100)"
-                />
-                <template #overlay>
-                  <NcMenu>
-                    <NcMenuItem class="!text-red-500 !hover:bg-red-50" @click="removeCollaborator(collab.id)">
-                      <MaterialSymbolsDeleteOutlineRounded />
-                      Remove user
-                    </NcMenuItem>
-                  </NcMenu>
-                </template>
-              </NcDropdown>
-            </div>
-          </td>
-          <td class="w-1/5 padding"></td>
-          <td class="w-1/5 padding"></td>
-        </tr>
-      </tbody>
-    </table>
+          </div>
+          <!-- <div class="w-2/5"></div> -->
```

Comment:
why comment it ? if not wanted just remove it

Reply:
same as above.